<ui version="4.0" >
 <class>NMGPreferencesBaseWidget</class>
 <widget class="QDialog" name="NMGPreferencesBaseWidget" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="widgetGeneral" >
      <attribute name="title" >
       <string>Genera&amp;l</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="1" >
        <widget class="QLineEdit" name="txtConfDirectoryPath" />
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="txtImagesDirectoryPath" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lblImagesDirectoryPath" >
         <property name="text" >
          <string>Images directory path</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="labelConfigurationDirectoryPath" >
         <property name="text" >
          <string>Configuration directory path</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lblLogFilePath" >
         <property name="text" >
          <string>Log file path</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="txtLogFilePath" />
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="txtResultDirectoryPath" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="lblResultDirectoryPath" >
         <property name="text" >
          <string>Result directory path</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="btnConfDirectoryPath" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QPushButton" name="btnImagesDirectoryPath" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QPushButton" name="btnLogFilePath" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QPushButton" name="btnResultDirectoryPath" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetPlugins" >
      <attribute name="title" >
       <string>Pl&amp;ugins</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="lblDefaultPluginPath" >
           <property name="text" >
            <string>Default plugins path</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtDefaultPluginPath" />
         </item>
         <item>
          <widget class="QPushButton" name="btnDefaultPluginPath" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="btnGroupPLugins" >
         <property name="title" >
          <string>Loading policy</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="gtnLoadAllPlugins" >
            <property name="text" >
             <string>Load all plugins</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QRadioButton" name="btnSelectPluginsToLoad" >
              <property name="text" >
               <string>Load selected plugins</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSelectPlugins" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>21</width>
              <height>31</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPageConnection" >
      <attribute name="title" >
       <string>Connection</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>NetMeter server address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtNetMeterAddress" >
           <property name="text" >
            <string>localhost</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="textLabel1" >
           <property name="text" >
            <string>NetMeter server port</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxNetMeterPort" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum" >
            <number>1024</number>
           </property>
           <property name="maximum" >
            <number>65000</number>
           </property>
           <property name="value" >
            <number>6122</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPageStations" >
      <attribute name="title" >
       <string>Stations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QPushButton" name="btnManageStations" >
           <property name="text" >
            <string>&amp;Manage Stations</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>137</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="maximumSize" >
        <size>
         <width>70</width>
         <height>24</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="maximumSize" >
        <size>
         <width>70</width>
         <height>24</height>
        </size>
       </property>
       <property name="text" >
        <string>C&amp;ancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>btnSelectPluginsToLoad</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnSelectPlugins</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>NMGPreferencesBaseWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>NMGPreferencesBaseWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
